Lib/test/test_zipimport.py

```diff
@@ -446,6 +446,30 @@ def testNamespacePackage(self):
         mod = importlib.import_module('.'.join((subpkg, TESTMOD + '3')))
         self.assertEqual('path1.zip', mod.__file__.split(os.sep)[-4])
 
+    def testImportSubmodulesInZip(self):
+        with ZipFile(TEMP_ZIP, "w") as z:
+            z.writestr("a/__init__.py", b'')
+            z.writestr("a/b/c/__init__.py", b'def foo(): return "foo"')
+
+        importer = zipimport.zipimporter(TEMP_ZIP + "/a/")
+        spec = importer.find_spec("a.b")
+        self.assertEqual(spec.loader, None)
+        self.assertEqual(spec.submodule_search_locations,
+                         [(TEMP_ZIP + "/a/b").replace("/", zipimport.path_sep)])
+        self.assertRaises(zipimport.ZipImportError, importer.get_code, "a.b")
+        self.assertEqual(importer.get_data("a/b/"), b"")
```

Comment:
I don't know if this is a necessary test case for `get_data`, maybe directory path will be never passed to it.